### PR TITLE
next/375/20240408/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #10774
 - #10777
 - #10782

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 19965

File: src/decode-tcp.h [hunk 1]
```diff
@@ -168,11 +168,11 @@ typedef struct TCPVars_
     TCPOpt tfo;         /* tcp fast open */
 } TCPVars;
 
-#define CLEAR_TCP_PACKET(p) {   \
-    (p)->level4_comp_csum = -1; \
-    PACKET_CLEAR_L4VARS((p));   \
-    (p)->tcph = NULL;           \
-}
+#define CLEAR_TCP_PACKET(p)                                                                        \
```
Comment (lukashino) [inline src/decode-tcp.h:171]: do you know why are the backslashes so offseted?

**Review by lukashino — APPROVED** [review]
If the offset of the backslashes are considered as a desired change then it looks good to me.

**Comment by victorjulien** [comment]
> If the offset of the backslashes are considered as a desired change then it looks good to me.

Yeah, this is intentional. We are moving to eventual clang-format compliance this way.

**Pull request merged**